University-advisor/Forms/ForgotPasswordForm.cs

```diff
@@ -38,23 +39,23 @@ private void ResetPasswordButton_Click(object sender, EventArgs e)
                 try
                 {
                     new PasswordResetService(usernameText.Text, emailText.Text).Reset();
-                    Logger.Log("Password successfully changed.");
-                    MessageBox.Show("Password successfully changed. Check your inbox. ");
+                    Logger.Log(Messages.passwordUpdateSuccess);
+                    MessageBox.Show(Messages.passwordUpdateSuccessInbox);
 
                     var loginForm = new LoginForm();
                     Hide();
                     loginForm.Show();
                 }
                 catch
                 {
-                    Logger.Log("Password cannot be changed.");
-                    MessageBox.Show("Password cannot be changed.");
+                    Logger.Log(Messages.passwordUpdateFailed);
+                    MessageBox.Show(Messages.passwordUpdateFailed);
                 }
             }
             else
             {
-                Logger.Log("User attempted to change password");
-                MessageBox.Show("User with provided information does not exist.");
+                Logger.Log(Messages.userAttemptedToChangePassword);
+                MessageBox.Show(Messages.userDontExist);
```

Comment:
Maybe we should add message to Logger that the attempt was not successfull?

Reply:
Okay, fixed that. Let me know if you have more suggestions